emoji/src/EmojiSearchTable/EmojiSearchTable.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+
+import style from './style.module.css';
+import EmojiSearch from '../EmojiSearch';
+import EmojiTable from '../EmojiTable';
+import emojiList from '../emojiList.json';
+
+export default function EmojiSearchTable(props) {
+    const [value, setValue] = useState('');
+
+    const saveValue = (event) => {
```

Comment:
Try to use [useCallback](https://ru.reactjs.org/docs/hooks-reference.html#usecallback) hook for every function declared inside components to avoid redundant re-renders